src/framework/uicomponents/view/popupwindow/popupwindow_qquickview.cpp

```diff
@@ -40,6 +40,8 @@ PopupWindow_QQuickView::~PopupWindow_QQuickView()
 
 void PopupWindow_QQuickView::init(QQmlEngine* engine, bool isDialogMode, bool isFrameless)
 {
+    QQuickWindow::setDefaultAlphaBuffer(!isDialogMode);
```

Comment:
Why is this necessary? And is it valid to call it at this point? The [documentation](https://doc.qt.io/qt-6/qquickwindow.html#setDefaultAlphaBuffer) might be interpreted as that it is only valid to call this before the very first QQuickWindow has been created, but maybe that's not what they mean.